Kconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2023 Renesas Electronics Corporation.
+# Copyright (C) 2023 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+config DOMD_ID
+	int "Dom D id"
+	default 1
+
+config SM_VCHAN_PATH
+	string "Path to the SM vchan"
+	default "/vchan/sm"
+
+config CONNECTION_TIMEOUT_SEC
+	int "Vchan connection timeout"
+	default 5
+
+config NODE_ID
+	string "Node id"
+	default "NODE_0"
+
+config NODE_TYPE
+	string "Nodde type"
+	default "NODE_TYPE1"
+
+config NUM_CPU
```

Comment:
Num CPU, total RAM, partition size should be detected automatically I guess. We can keep as internal const right now.

Reply:
lets currently keep it on config, because it could be different in Gen3 and Gen4 HW